scripts/cnv_wdl/somatic/README.md

```diff
@@ -0,0 +1,58 @@
+## Running the Somatic CNV WDL
```

Comment:
This file was moved, right?  Is there anything in the content (other than renamed workflow) that is different?  I did not see anything...